src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,86 @@
+import classNames from 'classnames';
+import React, { useMemo } from 'react';
+import { useTodos } from '../../TodosContext';
+import { Status } from '../../types/Status';
+
+export const Footer: React.FC = () => {
+  const {
+    todos,
+    clearCompleted,
+    selectedStatus,
+    setSelectedStatus,
+    notCompletedTodos,
+  } = useTodos();
+
+  const onFilterSelect = (status: Status) => () => {
+    setSelectedStatus(status);
+  };
+
+  const areCompletedTodos = useMemo(() => {
+    return todos.some(todo => todo.completed);
+  }, [todos]);
```

Comment:
why do you have this logic here if a lot of the same thing calculates App component?